src/porepy/models/constitutive_laws.py

```diff
@@ -804,23 +688,13 @@ def fluid_viscosity(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
         return Scalar(self.fluid.viscosity(), "viscosity")
 
 
-class SecondOrderTensorUtils:
-    basis: Callable[[Sequence[pp.GridLike], int], list[pp.ad.SparseArray]]
-    """Basis for the local coordinate system. Normally set by a mixin instance of
-    :class:`porepy.models.geometry.ModelGeometry`.
-
-    """
+class SecondOrderTensorUtils(PorePyModel):
     specific_volume: Callable[
```

Comment:
I know this is defined as a constitutive law, but it also in essense geometry, and specefically, it should be independent of the type of physics considered (mass or momentum balance etc.). Is it meaningful to include it in the PorePyModelProtocol?

The same reasoning, yes or no, can probably be used for a couple of other quantities, suhc as isotropic_second_order_tensor below

Reply:
Added specific_volume, aperture and isotropic_second_order_tensor.